docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -1,184 +1,136 @@
-service "php-apache" exposed
-```
-
-Now, we will wait some time and verify that both the replication controller and the service were correctly created and are running. We will also determine the IP address of the service:
+First, we will start a deployment running the image and expose it as a service:
 
 ```shell
-$ kubectl get pods
-NAME               READY     STATUS    RESTARTS   AGE
-php-apache-wa3t1   1/1       Running   0          12m
-
-$ kubectl describe services php-apache | grep "LoadBalancer Ingress"
-LoadBalancer Ingress:	146.148.24.244
-```
-
-We may now check that php-apache server works correctly by calling `curl` with the service's IP:
-
-```shell
-$ curl http://146.148.24.244
-OK!
-```
-
-Please notice that when exposing the service we assumed that our cluster runs on a provider which supports load balancers (e.g.: on GCE).
-If load balancers are not supported (e.g.: on Vagrant), we can expose php-apache service as ``ClusterIP`` and connect to it using the proxy on the master:
-
-```shell
-$ kubectl expose rc php-apache --port=80 --type=ClusterIP
-service "php-apache" exposed
-
-$ kubectl cluster-info | grep master
-Kubernetes master is running at https://146.148.6.215
-
-$ curl -k -u <admin>:<password> https://146.148.6.215/api/v1/proxy/namespaces/default/services/php-apache/
-OK!
+$ kubectl run php-apache --image=gcr.io/google_containers/hpa-example --requests=cpu=200m --expose --port=80
+service "php-apache" created
+deployment "php-apache" created
 ```
 
 ## Step Two: Create horizontal pod autoscaler
 
-Now that the server is running, we will create a horizontal pod autoscaler for it.
-To create it, we will use the [hpa-php-apache.yaml](/docs/user-guide/horizontal-pod-autoscaling/hpa-php-apache.yaml) file, which looks like this:
-
-```yaml
-apiVersion: extensions/v1beta1
-kind: HorizontalPodAutoscaler
-metadata:
-  name: php-apache
-  namespace: default
-spec:
-  scaleRef:
-    kind: ReplicationController
-    name: php-apache
-    namespace: default
-  minReplicas: 1
-  maxReplicas: 10
-  cpuUtilization:
-    targetPercentage: 50
-```
-
-This defines a horizontal pod autoscaler that maintains between 1 and 10 replicas of the Pods
-controlled by the php-apache replication controller we created in the first step of these instructions.
+Now that the server is running, we will create the autoscaler using
+[kubectl autoscale](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/docs/user-guide/kubectl/kubectl_autoscale.md).
+The following command will create a horizontal pod autoscaler that maintains between 1 and 10 replicas of the Pods
+controlled by the php-apache deployment we created in the first step of these instructions.
 Roughly speaking, the horizontal autoscaler will increase and decrease the number of replicas
-(via the replication controller) so as to maintain an average CPU utilization across all Pods of 50%
+(via the deployment) so as to maintain an average CPU utilization across all Pods of 50%
```

Comment:
(via the deployment) to maintain an average CPU utilization across all Pods close to 50%